extensions/wikia/ArticleComments/classes/ArticleComment.class.php

```diff
@@ -1114,19 +1114,19 @@ static public function addArticlePageToWatchlist( $comment ) {
 	 * @return Bool true -- because it's a hook
 	 */
 	static public function watchlistNotify( RecentChange &$oRC ) {
-		global $wgEnableGroupedArticleCommentsRC;
-		wfProfileIn( __METHOD__ );
+		$app = F::app();
```

Comment:
Given `$app` isn't used directly for anything else but `wg`, this could be:
```php
$wg = F::app()->wg;
```
and we could reduce line length on subsequent lines.